### DISCUSSION TITLE
Forcing widget refresh on frontend & the backbone view reference

### DISCUSSION THREAD

**Discussion by thsbrown** [elementor/elementor#15476] [Developers Community]
I'm curious if there is a relatively simple way to force refresh/reload a widget via ajax?

After doing extensive research into the code base it looks, like the `resetStyle()` method bound to the widget context menu / backbone.view would work for me, unfortunately that would require the backbone view reference.

Is there any easy way to get a backbone.view reference to a widget via javascript? If not is there any other way I might be able to refresh/reload my widget another way?

The reason this is all necessary is that elementor currently does not fire the `element_ready ` action when dragging and dropping on a widget in the same location [see here](https://github.com/elementor/elementor/issues/7234), so some of my code is failing to reinitialize despite something changing when the dragging and dropping on same location occurs.

**Code For Reference**
```
    $( window ).on( 'elementor/frontend/init', function() {
            elementorFrontend.hooks.addAction( 'frontend/element_ready/double-subtitled-video.default', (scope,jquery) => {
                //videojs fails initialization when dragging and dropping in same location due to above action not firing
                new VideoPlayer(scope[0].getElementsByTagName("video").item(0));
            } );
    } );
```

**Comment by dbjpanda** [comment]
I too wanted similar functionality. I wanted to refresh a widget say post widget on trigger of an action from my custom widget.

**Comment by harrywhitedev** [comment]
Elementor Documentation for developers is awful.
